libraries/botbuilder-ai/src/luis-client/luisClient.ts

```diff
@@ -0,0 +1,277 @@
+/**
+ * @module botbuilder-ai
+ */
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+/**
+ * NOTE: This class was auto generated by OpenAPI Generator (https://openapi-generator.tech)
+ * and was manually modified to make it compliant with the current implementation of the library.
+ */
+
+import Request = require('request');
+import * as HttpStatus from 'http-status-codes';
+
+/* tslint:disable:no-unused-locals */
+import { LuisResult } from './model/luisResult';
+
+import { ObjectSerializer, Authentication, VoidAuth } from './model/models';
+import { ApiKeyAuth } from './model/models';
+
+let luisVersion = '/luis/v2.0';
+
+export interface PredictionResolveOptionalParams {
+    /**
+     * The timezone offset for the location of the request.
+     */
+    timezoneOffset?: number;
+    /**
+     * If true, return all intents instead of just the top scoring intent.
+     */
+    verbose?: boolean;
+    /**
+     * Use the staging endpoint slot.
+     */
+    staging?: boolean;
+    /**
+     * Enable spell checking.
+     */
+    spellCheck?: boolean;
+    /**
+     * The subscription key to use when enabling bing spell check
+     */
+    bingSpellCheckSubscriptionKey?: string;
+    /**
+     * Log query (default is true)
+     */
+    log?: boolean;
+
+    customHeaders: {headers: {[name: string]: string}};
+}
+
+export enum LuisApikeys {
+    apiKeyHeader,
+}
+
+export class LuisClient {
+    protected _basePath: string = '';
+    protected _useQuerystring: boolean = false;
```

Comment:
This defaults to false, is not exposed in the constructor and has no docstring; what's the story around this member for users?

Reply:
This is another auto-generated member. As this wasn't reference in any other class (nor in the .LuisClient), we just removed it.